symbols/overlay10.yml

```diff
@@ -219,6 +219,29 @@ overlay10:
         
         r0: move_id
         return: anim_ent_ptr (This might be a mistake? It seems to be an integer, not a pointer)
+    - name: DrawTeamStats
+      address:
+        EU: 0x22C1328
+      description: "Handles creating the windows, sprites, etc. for the team stats top screen display."
```

Comment:
Missing parameters and return value. It's a good idea to specify them, even if their type and purpose is unknown.

Reply:
My rule is usually to match what's in the headers. For cases where we're _completely_ unsure of anything, not having params in the descriptions corresponds to not having a prototype in the headers at all.

In this case though, we have a prototype that at least gives us information on the number of parameters and the type widths. So in that case, it's good to match that in the headers. Something like:

```
r0: ?
r1: ?
r2: ?
r3: ?
return: ?
```